### DISCUSSION TITLE
Connecting to Moonraker using IPv6

### DISCUSSION THREAD

**Discussion by flajzi** [KlipperScreen/KlipperScreen#380] [Q&A]
Hi folks,
I'm running Moonraker on interface listening on IPv6 address. I'm not able to connect to Moonraker using KlipperScreen, I've tried "::1" (loopback address - Moonraker and KlipperScreen are running on same machine), also have tried full IPv6 global address of ethernet adapter and have no success.
Unfortunately Moonraker cannot listen on IPv4 and IPv6 simultaneously, so I cannot run Moonraker on IPv4 and use 127.0.0.1 as usually.
Can KipperScreen handle IPv6 at all?

**Comment by orobardet** [comment]
Hi, 
Did you try with `[::1]` (with the [])? In URI, this is the way to set an ipv6 has host part of the URL (has an ipv6 contains ':', that is normally used as the port separator).
